.circleci/config.yml

```diff
@@ -5,9 +5,17 @@ orbs:
   gcp-cli: circleci/gcp-cli@1.8.4
 
 commands:
-  docker-login:
+  login-docker:
```

Comment:
why reverse the name?  `docker-login` makes more sense since it's calling `docker login`

Reply:
hmm: it's just consistent with our naming in the determined repo.

Reply:
oh, ok.  works for me.